packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -76,7 +92,25 @@ const ItemGroup: React.FC<React.PropsWithChildren<WithStyles<typeof styles> & Me
     }
 
     if (!menuOpen) {
-        displayedTitle = shortTitle || getInitials(title);
+        let titleAsString: string;
+
+        if (!shortTitle) {
+            titleAsString = getInitials(title);
+        } else if (typeof shortTitle === "string") {
+            titleAsString = shortTitle;
+        } else if (isFormattedMessage(shortTitle)) {
+            titleAsString = intl.formatMessage(shortTitle.props);
+        } else {
+            titleAsString = getInitials(title);
+            console.warn("The prop 'shortTitle' must be either a string or a FormattedMessage to take effect.");
+        }
+
+        if (titleAsString.length > 3) {
+            titleAsString = getInitials(title);
+            console.warn("The prop 'shortTitle' must be no longer than 3 characters to take effect.");
+        }
+
+        displayedTitle = titleAsString;
```

Comment:
This isn't part of this PR, right?

Reply:
how is this a change? 🤔 Maybe because of a rebase.